ghost/core/core/server/adapters/storage/S3Storage.ts

```diff
@@ -48,6 +56,8 @@ export interface S3StorageOptions {
     sessionToken?: string;
     tenantPrefix?: string;
     s3Client?: S3Client;
+    multipartThreshold?: number; // File size threshold for multipart upload (default: 10MB)
+    partSize?: number; // Size of each part for multipart upload (default: 10MB)
```

Comment:
I think we can remove comments here, we can make the names more explicit if we're worried about losing context - maybe `multipartUploadThresholdBytes` and `multipartChunkSizeBytes` would capture everything?